### ISSUE TITLE
Code lab review linked from "Identify resources loaded from the network"

### ISSUE THREAD

**Issue by alankent** [GoogleChrome/web.dev#225]
I did the code lab regarding watching network traffic. Generally fine, but here is feedback, including inconsistent results between the text and the results I observed. It was not that confusing to me, but if someone is learning this for the first time getting the text accurate I think is worthwhile.

* The "Show Live" icon was different to Glitch. A pain I know whenever Glitch updates their UI.
* “Open the DevTools and and select…”  -- duplicate “and”, plus in my menus it is called “Developer Tools” not DevTools.
* The instructions said you open the page, then open DevTools. When I did that it did *not* show the network trace as described. I had to reload the page to get the network traffic shown. (It was not hard to work out, but for a non-expert that might get confused.)
* The text says I will get requests for style.css and app.js, but instead I got app.js, inject.js, and enwiki-1.5x.png. I did not see any request for style.css. This is the most serious issue on the page for me because a newcomer might be confused.

**Comment by robdodson** [comment]
@jeffposnick I think this is referring to this codelab: https://web.dev/reliable/identify-resources-via-network-panel/codelab-explore-network-panel

Are you able to take a look and verify/fix the issues Alan raised?

**Comment by jeffposnick** [comment]
Sure thing—thanks for raising this, Alan!

**Comment by jeffposnick** [comment]
I believe you fixed some of these issues in https://github.com/GoogleChrome/web.dev/pull/434, @robdodson.

I've removed the `enwiki-15.x.png` favicon from the web app, which should eliminate one of the misleading entries. I did see `styles.css` and did not see `inject.js` in the list, so I think whatever might have been causing that network traffic back in November is no longer an issue.

I'm inclined to close this now, but let me know if you think there's anything that needs to be addressed beyond that.
